sdks/python/apache_beam/io/datastore/v1/datastoreio_test.py

```diff
@@ -118,6 +122,48 @@ def test_SplitQueryFn_with_query_limit(self):
       self.assertEqual(1, len(returned_split_queries))
       self.assertEqual(0, len(self._mock_datastore.method_calls))
 
+  def test_DatastoreWriteFn_with_emtpy_batch(self):
+    self.check_DatastoreWriteFn(0)
+
+  def test_DatastoreWriteFn_with_one_batch(self):
+    num_entities_to_write = _Mutate._WRITE_BATCH_SIZE * 1 - 50
+    self.check_DatastoreWriteFn(num_entities_to_write)
+
+  def test_DatastoreWriteFn_with_multiple_batches(self):
+    num_entities_to_write = _Mutate._WRITE_BATCH_SIZE * 3 + 50
```

Comment:
Will it be possible to add tests that run pipelines for read/write/delete that use DirectRunner and fake datastore ?